docs/networking/networking.md

```diff
@@ -103,3 +103,46 @@ If you disable the built-in CCM and do not deploy and properly configure an exte
 ## Nodes Without a Hostname
 
 Some cloud providers, such as Linode, will create machines with "localhost" as the hostname and others may not have a hostname set at all. This can cause problems with domain name resolution. You can run K3s with the `--node-name` flag or `K3S_NODE_NAME` environment variable and this will pass the node name to resolve this issue.
+
+## Multicluster CIDR (Experimental)
+
+:::info Version Gate
+
+Experimental as of v1.26.3+k3s1
+
+:::
+
+:::caution Warning
+The network policy controller could not work properly when this flag is enabled.
+:::
+
+From `v1.26` Kubernetes introduced Multicluster CIDR as an alpha feature. This feature can be enabled on K3s server with the `--multi-cluster-cidr` flag.
```

Comment:
maybe we could have a link to the KEP in our doc: https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/2593-multiple-cluster-cidrs